agent/mission_solver/__main__.py

```
@@ -0,0 +1,99 @@
+            print(f"日期格式错误: {args.date}，应为 YYYY-MM-DD")
+            sys.exit(1)
+
+    # 获取当前周常任务
+    print(f"=== 周常任务求解器 (region={args.region}) ===\n")
+
+    missions = get_current_missions(args.region, target_date)
+    if not missions:
+        print("未找到当前周的周常任务数据。")
+        print("请确认:")
+        print(f"  1. 数据文件存在: assets/resource/Chaldea/master_missions_{args.region}.json")
+        print(f"  2. 当前日期有对应的周常任务")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)